google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/impl/GooglePubSubSource.scala

```diff
@@ -33,14 +36,21 @@ private[pubsub] final class GooglePubSubSource(projectId: String,
       def fetch(implicit mat: Materializer): Unit = {
         import mat.executionContext
 
-        val req = if (httpApi.isEmulated) {
+        def pull(maybeAccessToken: Option[String]): Future[PullResponse] =
           httpApi
-            .pull(project = projectId, subscription = subscription, maybeAccessToken = None)
+            .pull(project = projectId,
+                  subscription = subscription,
+                  maybeAccessToken = maybeAccessToken,
+                  returnImmediately = returnImmediately,
+                  maxMessages = maxMessages)
```

Comment:
Not necessary to list the names of the arguments here. This is usually done when there are default values for arguments, and only some of them are passed from the call site.

Reply:
yea, Its not technically necessary. But I thought it was a little nicer, easier to review. But sure, i'll take it out.